### DISCUSSION TITLE
Auto Play Movie when you click "Play 'movie name'"

### DISCUSSION THREAD

**Discussion by heylucifurr** [DonutWare/Fladder#105] [Ideas]
### Summary

I notice that when you are in the movie poster or homepage, when you click on the Play movie, it doesnt play automatically. You cave to click play again.

### Basic Example

just like on native jellyfin client when you hit play in homepage or poster, it automatically play the movie once it loads the movie itself.

### Drawbacks

--

**Comment by PartyDonut** [comment]
This is probably more of a bug maybe?
The normal behaviour for Fladder is like you said. when you click play it should auto-start the video when done loading. 

If you could create a bug ticket with some more info maybe even a video that would be great 👍🏼 .

**Comment by heylucifurr** [comment]
Ohh I see... Ill try it on different browsers and phone and see how its behaving and report it later. 

Thanks! youre doing a great work out there!
